gammapy/data/data_store.py

```diff
@@ -304,6 +306,25 @@ def check_available_event_lists(self, logger=None):
 
         return file_available
 
+    def create_new_store_from_obs_table(self, obs_table, outdir):
```

Comment:
Maybe just "copy_obs" to be a bit shorter? (no strong opinion ... pick which one you like best)